src/platform/windows/window.rs

```diff
@@ -44,7 +44,7 @@ impl Window {
     {
         let mut w_attr = Some(w_attr.clone());
         let mut pl_attr = Some(pl_attr.clone());
-
+        
```

Comment:
That was in the [original `window.rs`](https://github.com/tomaka/winit/blob/master/src/platform/windows/window.rs#L47). I've removed it now.